docs/docs/getting-started/install.md

```diff
@@ -111,6 +111,14 @@ If you don't have a cloud subscription, you can try [MiniConstellation](first-st
 <tabs groupId="csp">
 <tabItem value="azure" label="Azure">
 
+The following resource providers need to be [registered](https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/resource-providers-and-types#register-resource-provider) in your subscription:
```

Comment:
> I haven't added instructions on how to register the resource providers, since I didn't exactly know where to put them.

IMO this link is sufficient.